bedrock/settings/static_media.py

```diff
@@ -349,8 +343,6 @@
     },
     'firefox_fx38_0_5_firstrun': {
         'source_filenames': (
-            'css/sandstone/sandstone.less',
-            'css/tabzilla/tabzilla-static.less',
```

Comment:
Gah. So now this page no longer has `tabzilla-static.less`, because it's not included in the `firefox` bundle (which this page inherits from `firefox/base.html`). I'll leave this one up to you - should `tabzilla-static.less` be included in the `firefox` bundle, or added a la carte in each page?